.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
```

Comment:
- [ ] Please the comments should be clear and easy to understand what do you mean by your code length if you mean to refer to a linter error it would be better to say please fix the linter error the length of each line in the code at most should be 30 letter.
![image](https://user-images.githubusercontent.com/74744787/160491872-735d736d-5fb1-4d54-8797-b6b6f9fcba16.png)

Reply:
Hi @MeqdamAlqudah ,the reason here. I am the one who added the linters and i don't have to modify the existed code except to give a comment.and that's why i am giving a comment on html and css not on linter configurations

Reply:
@freddykaberuka what do I mean here you didn't explain the problem in detail you can just make that comment about linters